### PR TITLE
Fix broken definitions file

### PR DESCRIPTION
**Pull request by BrennanConroy**
Fixes https://github.com/aspnet/AspNetCore/issues/5400

### CODE REVIEW

**Comment by BrennanConroy** [comment]
#### Description 
In some environments, consuming the `@aspnet/signalr-protocol-msgpack` npm package is not possible because it causes build failures.
 
#### Customer Impact 
Customers using the msgpack package already, can't upgrade from 1.0.0 to 1.1.0

#### Regression?
Yes, using the msgpack package worked before.
 
#### Risk
Very Low

**Review by mikaelm12 — COMMENTED** [review]
The change itself looks fine but I want to be sure that this actually fixes the issue since we've had multiple customers reporting issues with this. Let's verify this manually

**Comment by halter73** [comment]
Did you find a way to repro this without VS? Why didn't this type incompatibility get detected automatically?

**Comment by natemcmaster** [comment]
Please wait until https://github.com/aspnet/AspNetCore-Internal/issues/1610 is resolved before merging.

**Review by muratg — APPROVED** [review]

**Comment by jamshedd** [comment]
Approved for 2.2.2.

**Comment by BrennanConroy** [comment]
@mikaelm12 @halter73 Need review

I did manual verification for this and figured out a way to reproduce the error via command line, however it requires writing an angular app and making it build as part of our test runs which will require a lot of work.

I'll keep the original issue open and re-purpose it for adding automated testing so we can get the patch fix in asap

**Review by mikaelm12 — APPROVED** [review]

**Comment by BrennanConroy** [comment]
@natemcmaster Could you either merge this or get the bot to run :(

I looked at all the failures and they have issues associated with them

**Comment by natemcmaster** [comment]
For 2.2, we haven't finished fixing all tests on Azure Pipelines, so we don't yet-require the AspNetCore-ci check to pass. That requirement has only been applied to the master branch.

**Review by halter73 — APPROVED** [review]

File: src/SignalR/clients/ts/signalr-protocol-msgpack/src/MessagePackHubProtocol.ts [hunk 1]
```diff
@@ -27,7 +27,7 @@ export class MessagePackHubProtocol implements IHubProtocol {
 
     /** Creates an array of HubMessage objects from the specified serialized representation.
      *
-     * @param {ArrayBuffer | Buffer} input An ArrayBuffer containing the serialized representation.
+     * @param {ArrayBuffer | Buffer} input An ArrayBuffer or Buffer containing the serialized representation.
```
Comment (halter73) [inline src/SignalR/clients/ts/signalr-protocol-msgpack/src/MessagePackHubProtocol.ts]: @BrennanConroy Why didn't make input `string | ArrayBuffer | Buffer` and match IHubProtocol exactly? Why don't you get an error like `Type 'string | ArrayBuffer | Buffer' is not assignable to type 'ArrayBuffer | Buffer'`?

**Pull request merged**